### ISSUE TITLE
The requested module 'david' does not provide an export named 'getDependencies'

### ISSUE THREAD

**Issue by cedx** [cedx/gulp-david#16]
Using these tasks:

```javascript
const {series, src, task} = require('gulp');

let david;
task('deps:check', () => src('package.json').pipe(david()));
task('deps:import', () => import('@cedx/gulp-david').then(mod => david = mod.david));
task('deps', series('deps:import', 'deps:check'));
```

I get the following error:

```
$ gulp deps
$ gulp deps
[21:51:06] Using gulpfile [...]\gulpfile.cjs
[21:51:06] Starting 'deps'...
[21:51:06] Starting 'deps:import'...
[21:51:06] 'deps:import' errored after 33 ms
[21:51:06] file:///C:/Projets/sab/cli/node_modules/@cedx/gulp-david/lib/checker.js:1
import { getDependencies, getUpdatedDependencies } from 'david';
         ^^^^^^^^^^^^^^^
SyntaxError: The requested module 'david' does not provide an export named 'getDependencies'
    at ModuleJob._instantiate (internal/modules/esm/module_job.js:92:21)
    at async ModuleJob.run (internal/modules/esm/module_job.js:107:20)
    at async Loader.import (internal/modules/esm/loader.js:141:24)
[21:51:06] 'deps' errored after 39 ms
```

**Comment by cedx** [comment]
Only the “default export” is supported for CommonJS files or packages :cry:

**Comment by cedx** [comment]
Fixed in version 13.1.0.
